languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -134443,6 +134443,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <suggestion><match no="1" regexp_match="(?iu)s$" regexp_replace=""/></suggestion>
                 <example correction="PIB">Les <marker>PIBs</marker> des pays émergents sont croissants.</example>
             </rule>
+            <rule tags="picky"><!--rule12-->
+                <pattern>
+                    <token>et</token>
+                    <marker>
+                        <token regexp="yes" case_sensitive="yes" spacebefore="yes">al
+                            <exception scope="next">.</exception></token>
+                    </marker>
+                </pattern>
+                <message>Un point est requis.</message>
+                <suggestion>\2.</suggestion>
+                <example correction="al.">Docteur Hun, Docteur Deuh et <marker>al</marker> l'ont démontré.</example>
+                </rule>
```

Comment:
It is always good to add some correct examples (al-Andalus, al dente...) to make sure that the problem is solved.

Reply:
Done, excellent idea!